src/apiCall.js

```diff
@@ -175,6 +175,18 @@ module.exports = function apiCall(file, apiPort, itApi = it) {
             }
           }
         } catch (err) {
+          if (req.after && _.isFunction(req.after)) {
+            req.after.call(req, {
+              apiPort,
+              specs: config.apiCalls.swagger,
+              res: {},
+              expect,
+              op: operations[req.call],
+              body: allReqData,
+              error: err,
```

Comment:
Can we update the readme with this parameter?

Reply:
Also, would it be possible to write a unit test that verifies that the after call is made when there is an exception?

Reply:
Also, in the readme, be sure to document that the `res` field will be an empty object. Perhaps it should be `null` or even undefined?

Reply:
updated the readme only for now